src/material-examples/sidenav-autosize/sidenav-autosize-example.html

```diff
@@ -0,0 +1,16 @@
+<mat-sidenav-container class="example-container" autosize>
```

Comment:
Can you change these to the drawer variants? I only want people to use sidenav for fullscreen layouts